Code/GraphMol/Chirality.cpp

```diff
@@ -2376,6 +2376,10 @@ void legacyStereoPerception(ROMol &mol, bool cleanIt,
   bool hasStereoBonds = false;
   for (auto bond : mol.bonds()) {
     if (cleanIt) {
+      if (bond->hasProp(common_properties::_CIPCode)) {
+        bond->clearProp(common_properties::_CIPCode);
+      }
+
```

Comment:
I think this can just be:
```suggestion
      bond->clearProp(common_properties::_CIPCode);
```